examples/rapids-image/app-image-config-input.json

```diff
@@ -0,0 +1,16 @@
+{
+    "AppImageConfigName": "custom-rapids-image-config",
+    "KernelGatewayImageConfig": {
+        "KernelSpecs": [
+            {
+                "Name": "rapids",
```

Comment:
I don't see a kernel with this name inside the RAPIDS image. Looks like the kernel name should be `python3`

```
docker run -it rapidsai/rapidsai bash
(...)
(rapids) root@f5c87b9f6dc6:/rapids/notebooks# jupyter-kernelspec list
Available kernels:
  python3    /opt/conda/envs/rapids/share/jupyter/kernels/python3
```

Reply:
can display name be different?